src/api/TestsAPI.ts

```diff
@@ -0,0 +1,21 @@
+
+import express from "express"
+
+import { SessionController } from "../controller/SessionController";
+
+import bodyParser from "body-parser";
+
+const sessionController = new SessionController
+
+export const testsApp = express();
```

Comment:
Por que o nome `testApp`? Pelo que entendi serve só para validar o token certo? Se for isso não faria mais sentido utilizar `validatorApp`?

Reply:
solved...